protocol/v2/ssv/validator/committee.go

```diff
@@ -177,6 +196,13 @@ func (c *Committee) StartDuty(logger *zap.Logger, duty *spectypes.CommitteeDuty)
 
 	}
 
+	// Stop all expired committee runners, when new runner is created
+	go func() {
+		if err := c.stopExpiredRunners(logger, duty.Slot); err != nil {
+			logger.Error("couldn't stop expired committee runners", zap.Uint64("current_slot", uint64(duty.Slot)), zap.Error(err))
```

Comment:
since this func no longer stops anything but just removes from maps, i think we should rename it to and any mention about it from "stop" to "prune"